### PR TITLE
added search icon

### PR DESCRIPTION
**Pull request by Anjali48-hub**
Targets #{ISSUE_ID}

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 DB_PORT=3306
-DB_DATABASE=homestead
-DB_USERNAME=homestead
-DB_PASSWORD=secret
+DB_DATABASE=portal
+DB_USERNAME=root
+DB_PASSWORD=
 
 BROADCAST_DRIVER=log
 CACHE_DRIVER=file
```
Comment (gautam071) [inline .env.example:18]: I think we no need these changes

Comment (gautam071) [reply .env.example:18]: revert it back

**Review by gautam071 — CHANGES_REQUESTED** [review]
Left a comment

**Comment by Ayush8923** [comment]
@Anjali48-hub Please fix CI checks also.

**Comment by gautam071** [comment]
Please fix the CI Checks as well

**Pull request merged**